remixer_example/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I was thinking gitignoring it after adding it would do the trick so git would ignore it, but I think it does nothing *sigh* So I have to manually remember to never add this file.